src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -110,10 +111,20 @@ public function action_handle_verify_webhooks() : void {
 	 * Handle stripe errors into the admin UI.
 	 *
 	 * @since 5.3.0
-	 * @since TBD   Added check for ajax call.
+	 * @since TBD   Added check for ajax call, and additional logic to only run logic on checkout page and when Stripe is connected.
 	 */
 	public function handle_stripe_errors() {
 
+		// Only run this code when you are on the Stripe Payment page/tab.
+		if ( ! tribe( Settings::class )->is_on_tab_section( 'payments', 'stripe' ) ) {
+			return;
+		}
+
+		// Only run the logic if you are on the checkout page, or you are connected.
+		if ( ! tribe( Module::class )->is_checkout_page() || ! tribe( Merchant::class )->is_connected() ) {
```

Comment:
```suggestion
		if ( ! tribe( Module::class )->is_checkout_page() ) {
```
merchant one should not be needed here, let's combine the above one within the same if logic.